ga4gh/datarepo.py

```diff
@@ -345,6 +345,14 @@ def verify(self):
                     reference.getLocalId())
         for dataset in self.getDatasets():
             print("Verifying Dataset", dataset.getLocalId())
+            reference = self.getReferenceSets()[0].getReferences()[0]
```

Comment:
This is broken --- you need to get the referenceSet for each of the FeatureSets. No reason to think that they'd all be the same.